frontend/benefit/applicant/public/locales/en/common.json

```diff
@@ -159,5 +159,36 @@
     "linkText": "hakemiseen vaadittaviin tietoihin ja tarvittaviin liitteisiin",
     "newApplicationBtnText": "Tee uusi hakemus"
   },
-  "menuToggleAriaLabel": "Menu"
+  "menuToggleAriaLabel": "Menu",
+  "form": {
+    "validation": {
+      "number": {
+        "max": "Maximum value is {{max}}",
+        "min": "Value must be at least {{min}}",
+        "required": "This field is required"
+      },
+      "string": {
+        "email": "This field must be a valid email",
+        "max": "This field can be up to {{max}} characters long",
+        "min": "This field must be at least {{min}} characters long",
+        "required": "This field is required",
+        "url": "This field must be a valid URL",
+        "date": "This field must be a valid date",
+        "time": "Time must be given in the following format hh:mm",
+        "positiveNumber": "This field must be a valid number"
+      },
+      "date": {
+        "required": "This field is required",
+        "mustNotInThePast": "This date must not be in the past",
+        "min": "This date must be earliest {{min}}",
+        "max": "This date must be latest {{max}}",
+        "format": "This date is incorrect"
+      },
+      "time": {
+        "required": "This field is required",
```

Comment:
`required` seems to be common property, should we move it a step higher?

Reply:
yes, it was from another projects, as they have already all translations. probably the idea was that the message for different types might be different. in practice not, so i fixed.